src/vast_tracker.js

```diff
@@ -319,19 +319,30 @@ export class VASTTracker extends EventEmitter {
     }
   }
 
+  /**
+   * Send a request to the URI provided by the VAST <Error> element.
+   * @param {Object} [macros={}] - An optional Object containing macros and their values to be used and replaced in the tracking calls.
+   *
+   * @param {Boolean} [isCustomCode=false] - Flag to allow custom values on error code.
+   */
+
+  error(macros = {}, isCustomCode = false) {
+    this.trackURLs(this.ad.errorURLTemplates, macros, { isCustomCode });
+  }
+
   /**
    * Send a request to the URI provided by the VAST <Error> element.
    * If an [ERRORCODE] macro is included, it will be substitute with errorCode.
+   * This method is deprecated
    *
    * @param {String} errorCode - Replaces [ERRORCODE] macro. [ERRORCODE] values are listed in the VAST specification.
    * @param {Boolean} [isCustomCode=false] - Flag to allow custom values on error code.
    */
+
   errorWithCode(errorCode, isCustomCode = false) {
-    this.trackURLs(
-      this.ad.errorURLTemplates,
-      { ERRORCODE: errorCode },
-      { isCustomCode }
-    );
+    this.error({ ERRORCODE: errorCode }, isCustomCode);
+    //eslint-disable-next-line
+    console.log('errorWithCode : this method is deprecated');
```

Comment:
What do you think about this? Because it maybe be misunderstood at first glance when seen in the console.
```suggestion
    console.log('The method errorWithCode is deprecated, please use vast tracker error method instead');
```